build.sbt

```diff
@@ -201,5 +156,23 @@ lazy val docs = project
     micrositeGithubRepo       := "matryoshka",
     micrositeBaseUrl          := "/matryoshka",
     micrositeDocumentationUrl := "/matryoshka/docs/01-Index.html",
-    micrositeHighlightTheme   := "color-brewer"
-  )
+    micrositeHighlightTheme   := "color-brewer")
+
+/** A project just for the console.
+  * Applies only the settings necessary for that purpose.
+  */
+lazy val repl = crossProject dependsOn (core % "test->test;compile->compile") settings standardSettings settings (
```

Comment:
I have no idea – some of this (like this part) was taken from @paulp’s previous PR. I assumed it was intended to replace `console <<= console in Test`, which SBT had been spitting out warnings about for a while now.